src/index.html

```diff
@@ -2,16 +2,184 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset inputs">
+        <legend class="legend">Personal information</legend>
+        <label class="formField" for="surname">
+        Surname:
+          <input
+        type="text"
+        name="surname"
+        id="surname"
+        autocomplete="off"
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
            autocomplete="off"
```
<img width="998" alt="Screenshot 2022-11-24 at 10 49 12" src="https://user-images.githubusercontent.com/104769168/203735730-08a666c1-384e-4893-b450-1d8bf70183de.png">